non-wheel/active-jobs-dashboard/root.tpl

```diff
@@ -0,0 +1,64 @@
+<html>
+<head><title>Active Job Dashboard</title>
+<script type="text/javascript" src="jquery-latest.js"></script>
```

Comment:
I would suggest using the CDN here: <script
  src="https://code.jquery.com/jquery-3.2.0.min.js"
  integrity="sha256-JAW99MJVpJBGcbzEuXk4Az05s/XyDdBomFqNlM3ic+I="
  crossorigin="anonymous"></script>